lib/storage/src/content_manager/toc/transfer.rs

```diff
@@ -0,0 +1,15 @@
+use collection::operations::types::{CollectionError, CollectionResult};
+use collection::shards::transfer::ShardTransferConsensus;
+
+use crate::dispatcher::Dispatcher;
+
+impl ShardTransferConsensus for Dispatcher {
+    fn consensus_commit_term(&self) -> CollectionResult<(u64, u64)> {
+        self.consensus_state()
+            .map(|consensus| {
+                let state = consensus.hard_state();
+                (state.commit, state.term)
+            })
+            .ok_or_else(|| CollectionError::service_error("Consensus is not available"))
+    }
+}
```

Comment:
The trait will eventually get specialized functions for operations we need on consensus. They'll be implemented here where we have access to the consensus proposal channel and consensus state through `Dispatcher`.